coder_sniffer/Drupal/Test/NamingConventions/ValidVariableNameUnitTest.inc

```diff
@@ -2,3 +2,25 @@
 
 $UppperCamelName = 0;
 $lowerCamelName = 1;
+
+class TestAnnotationProperties extends Plugin {
+
+  /**
+   * The entity_type.
+   *
+   * @var string
+   */
+  public $entity_type;
+
+}
+
+class TestAnnotationProperties2 extends ViewsPluginAnnotationBase {
+
+  /**
+   * The entity_type.
+   *
+   * @var string
+   */
+  public $entity_type;
+
+}
```

Comment:
we are missing a test case for AnnotationInterface, right?

Reply:
You're right. I added a test case for AnnotationInterface.